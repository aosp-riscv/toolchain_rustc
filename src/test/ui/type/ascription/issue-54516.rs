use std::collections::BTreeMap;

fn main() {
    println!("{}", std::mem:size_of::<BTreeMap<u32, u32>>());
<<<<<<< HEAD   (086005 Importing rustc-1.38.0)
    //~^ ERROR expected token: `,`
=======
    //~^ ERROR expected one of
>>>>>>> BRANCH (8cd2c9 Importing rustc-1.39.0)
}
