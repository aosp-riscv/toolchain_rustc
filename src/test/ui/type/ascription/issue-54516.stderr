<<<<<<< HEAD   (086005 Importing rustc-1.38.0)
error: expected token: `,`
  --> $DIR/issue-54516.rs:4:58
   |
LL |     println!("{}", std::mem:size_of::<BTreeMap<u32, u32>>());
   |                            -                             ^ expected `,`
=======
error: expected one of `!`, `,`, or `::`, found `(`
  --> $DIR/issue-54516.rs:4:58
   |
LL |     println!("{}", std::mem:size_of::<BTreeMap<u32, u32>>());
   |                            -                             ^ expected one of `!`, `,`, or `::` here
>>>>>>> BRANCH (8cd2c9 Importing rustc-1.39.0)
   |                            |
   |                            help: maybe write a path separator here: `::`
   |
   = note: `#![feature(type_ascription)]` lets you annotate an expression with a type: `<expr>: <type>`
   = note: for more information, see https://github.com/rust-lang/rust/issues/23416

error: aborting due to previous error

